docs/tutorials/fundamentals/installation.md

```diff
@@ -78,13 +78,12 @@ Choose one of the options below to install napari as a Python package.
 ````{admonition} **1. From pip**
 :class: dropdown
 
-napari can be installed on most macOS, Linux, and Windows systems with Python
-3.7, 3.8, and 3.9 using pip:
```

Comment:
Removed reference to operating systems and python version since that information is already covered in the doc.